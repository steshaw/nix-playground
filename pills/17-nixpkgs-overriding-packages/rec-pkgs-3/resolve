#!/bin/sh

get_attrs() {
  nix-instantiate --eval --attr 'list' --json | jq -r '.[]'
}

attrs=$(get_attrs)

for attr in $attrs; do
  echo
  echo "$attr:"
  nix-instantiate --eval --strict --json --attr "$attr" | jq
done
