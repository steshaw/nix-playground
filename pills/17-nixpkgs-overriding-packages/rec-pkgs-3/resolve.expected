
pkgs-app:
{
  "a": 3,
  "b": 4,
  "c": 7,
  "d": 49,
  "f": 9
}

pkgs-fix:
{
  "a": 3,
  "b": 4,
  "c": 7,
  "d": 49,
  "f": 9
}
